Gemfile

```diff
@@ -2,15 +2,14 @@ source 'https://rubygems.org'
 ruby '2.3.3'
 
 gem 'rails', '~> 4.2.6'
-gem 'rails_12factor'
 
 # Use Postgresql as the database for Active Record
 gem 'haml'
 gem 'jquery-rails'
 gem 'jquery-ui-rails'
 gem 'mini_magick', '~> 3.6.0'
-gem 'nokogiri'
-gem 'pg'
+gem 'nokogiri', '~> 1.8.1'
```

Comment:
nokogiri gets a few security advisories lately -- if we pin it, deppbot won't send up security upgrades anymore.

Reply:
oh but with the squiggle-rocket operator the minor-minor version is unconstrained, so it can freely upgrade to any version that is `1.8.x` where `x` - so the patch version is still covered ✅

Reply:
I think of `~>` as meaning "equal to or greater than the last digit"